students/13th/bogyeongkim/login.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Instagram</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/login.css" rel="stylesheet" type="text/css" />
+     <link rel="shortcut icon" href="img/instagram.png"/>
+  </head>
+  <body>
+    <div class="total-container">
+    <header><img src="img/logo_text.png"alt="instagram"></header>
```

Comment:
```suggestion
    <header>
      <img src="img/logo_text.png"alt="instagram">
    </header>
```

한 줄에 쓰기 보다 줄을 띄어주시면 더 가독성이 좋을 것 같네요!

Reply:
수정완료~